must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -73,3 +73,8 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+# collect outpt for cephfilesystemsubvolumegroups.ceph.rook.io crd
```

Comment:
Q: why do we need to collect data for cephfilesystemsubvolumegroups separately?  Shouldn't this be part of above for loop?

Reply:
it can be. But for me, oc get cephfilesystemsubvolumegroups.ceph.rook.io didn't work even and saw that it's a crd. Hence have to collect it seprately.
Didn't get a new cluster to verify it, but it will be great if you can confirm what's the exact command for it.
We also had a discussion in gchats where neha mentioned that for her, `oc get cephfilesystemsubvolumegroups.ceph.rook.io ` is working fine.

Reply:
we should not treat this as a separate only can we add this with other resources like filesystem crd?

Reply:
>We also had a discussion in gchats where neha mentioned that for her, oc get cephfilesystemsubvolumegroups.ceph.rook.io is working fine

You need to pass namespace to oc command to get the subvolumegroup created `oc get cephfilesystemsubvolumegroups.ceph.rook.io -n openshift-storage`

Reply:
+1. It works for me. Agreed that we do not need to collect the crd

oc get cephfilesystemsubvolumegroups.ceph.rook.io -n openshift-storage
NAME                                                                                AGE
cephfilesystemsubvolumegroup-storageconsumer-da3ce6e4-e148-4ddd-9d1f-3ebf84bee072   33m

consumer
-----------
oc  get cephfilesystemsubvolumegroups.ceph.rook.io -n openshift-storage 
NAME                                                                                AGE
cephfilesystemsubvolumegroup-storageconsumer-da3ce6e4-e148-4ddd-9d1f-3ebf84bee072   33m